lib/fathead/python_pymongo/parse.py

```diff
@@ -0,0 +1,282 @@
+                    for item in tag.dd.contents:
+                        tag_description += str(item)
+                    tag_description = self._clean_html_tags(tag_description)
+
+                    output_line = ""
+                    abstract = '<section class="prog_container">' + \
+                                '<pre><code>' + tag_code + '</pre></code><p>' + \
+                                tag_description + '</p></section>'
+                    title = tag.dt["id"].replace('.', ' ')
+                    tag.decompose()
+
+                    filename_removed_dir = filename.replace('download/', '')
+                    headerlink = self.baseurl + filename_removed_dir + anchor_link
+                    # 1. Full article title
+                    output_line += title + "\t"
+                    # 2. Type of entry (A for article)
+                    output_line += "A" + "\t"
+                    # 3. For redirects only
+                    output_line += "" + "\t"
+                    # 4. Empty
+                    output_line += "" + "\t"
+                    # 5. Categories
+                    output_line += "" + "\t"
+                    # 6. Empty
+                    output_line += "" + "\t"
+                    # 7. Related topics
+                    output_line += "" + "\t"
+                    # 8. Empty
+                    output_line += "" + "\t"
+                    # 9. External Links (official site) [$url link_text]
+                    output_line += "" + "\t"
+                    # 10. Disambiguation pages only
+                    output_line += "" + "\t"
+                    # 11. Image
+                    output_line += "" + "\t"
+                    # 12. Abstract (Main content)
+                    output_line += abstract + "\t"
+                    # 13. Full URL to source (url with anchor)
+                    output_line += headerlink
+                    print(output_line)
+
+    def _clean_code_tags(self, code_html_to_clean):
+        """
+        Cleans <code> tags from html. Should only be used in 'code" section to prevent
+        <code> tags from doubling up within the <pre><code> section
+        :param code_html_to_clean: String of html to remove code tags
+        :return: html with removed code tags
+        """
+
+        html_soup_cleaner = BeautifulSoup(code_html_to_clean, 'html.parser')
+
+        # Remove code tags so only the ones we want are displayed
+        for tag in html_soup_cleaner.find_all("code"):
+            if tag.attrs:
+                del tag.attrs
+            del tag.name
+
+        # Reformat tables
+        for tag in html_soup_cleaner.find_all('table'):
+            if tag.attrs:
+                del tag.name
+                del tag.attrs
+            if tag.ul:
+                tag.contents = tag.ul.contents
+                tmp_tag = html_soup_cleaner.new_tag("span")
+                tmp_tag.attrs = {"class": "progr__sub"}
```

Comment:
It looks like there's still some lingering instance of `\nParameters\n` in the output, that aren't being wrapped in `<span class="prog__sub"` -- try some of the articles on Beta and you'll see